drivers/osdp/src/osdp_common.c

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2020 Siddharth Chandrasekaran <siddharth@embedjournal.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ctype.h>
+
+#include <device.h>
+#include <sys/crc.h>
+#include <logging/log.h>
+
+#include "osdp_common.h"
+
+LOG_MODULE_DECLARE(osdp, CONFIG_OSDP_LOG_LEVEL);
+
+void osdp_dump(const char *head, uint8_t *buf, int len)
+{
+	LOG_HEXDUMP_DBG(buf, len, head);
+}
+
+uint16_t compute_crc16(const uint8_t *buf, size_t len)
+{
+	return crc16_itu_t(0x1D0F, buf, len);
+}
+
+millis_t millis_now(void)
```

Comment:
Hmm, I am not convinced that we need yet another typedef here. Why not use int64_t directly?

Reply:
Done.